src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -160,10 +162,14 @@ public void stopAngledArm() {
         setMotor(ClimbConstants.STOP_SPEED, climbDynamicMotor);
     }
 
-    public void setMotor(double position, WPI_TalonFX motor) {
+    public void setMotor(double value, WPI_TalonFX motor) {
         if (state == ClimbSubsystemState.ENABLED) {
-            System.out.println("motor set: " + position);
-            motor.set(ControlMode.Position, position);
+            System.out.println("motor set: " + value);
+            if (Robot.isSimulation()) {
+                motor.getSimCollection().setIntegratedSensorVelocity((int) value);
```

Comment:
? this is setting a position value for velocity, which seems wrong.  I'd recommend instead getting the current position, comparing it with the target position and then setting velocity to a constant value of the correct sign (positive if the target position is larger, negative if the target position is smaller)